lang/en.json

```diff
@@ -399,26 +399,28 @@
             "no_items": "The items you've recently looked at on our site will appear below."
         },
         "wishlists": {
-            "heading": "Wish lists",
-            "tag_line": "Fill in the form below to create a wish list. Click the \"Save\" button when you're done.",
-            "new": "New wish List",
-            "you_have_none": "You have no wish lists, add one now.",
-            "empty_wishlist": "Your wish list is empty. When you add items to your wish list they will appear here.",
+            "heading": "Wish Lists",
+            "tag_line": "Fill in the form below to create a Wish List. Click the \"Create Wish List\" button when you're done.",
```

Comment:
should we add the work `new` to `Fill in the form below to create a Wish List.` 🍹 .

Reply:
sure